app/controllers/users_controller.rb

```diff
@@ -104,16 +104,16 @@ def public
       respond_to do |format|
         format.atom do
           @posts = Post.where(author_id: @user.person_id, public: true)
-                    .order('created_at DESC')
-                    .limit(25)
-                    .map {|post| post.is_a?(Reshare) ? post.absolute_root : post }
-                    .compact
+                   .order('created_at DESC')
+                   .limit(25)
+                   .map { |post| post.is_a?(Reshare) ? post.absolute_root : post }
+                   .compact
```

Comment:
I indent multiline method calls, if the space allows it to the first call, else by two spaces:

```ruby
foo.bar
     .baz
     .quux

long_call_that_will_have_many_methods_chained
  .foo
  .bar
  .baz